pwa/src/templates/EndpointsDetailTemplate/EndpointDetailsTemplate.tsx

```diff
@@ -40,9 +40,7 @@ export const EndpointDetailTemplate: React.FC<EndpointDetailsTemplateProps> = ({
   };
 
   const handleDeleteEndpoint = () => {
-    const confirmDeletion = confirm("Are you sure you want to delete this endpoint?");
-
-    confirmDeletion && deleteEndpoint.mutate({ id: endpointId });
+    deleteEndpoint.mutate({ id: endpointId });
```

Comment:
Following feedback goes for all implementations.

I think we can remove some of this boilerplate, because we've simplified this logic. Right now, we're creating a function, to hold a very basic piece of logic (the `deleteEndpoint.mutate(...)` is the only logic). Afterwards, we're passing this function to the `FormHeaderTemplate` component.

However, we could just simply pass the `deleteEndpoint.mutate(...)` directly, instead of wrapping it first, like so:

```
<FormHeaderTemplate
  ...
  handleDelete={() => deleteEndpoint.mutate({ id: endpointId })}}
  ...
/>
```

This way, this function declaration can be removed:

```
const handleDeleteEndpoint = () => {
  deleteEndpoint.mutate({ id: endpointId });
};
```